### PR TITLE
TT-11522 Update NOTES.txt

### PR DESCRIPTION
**Pull request by caroltyk**
## Description

Update control plane installation notes for

1. Explicit how to set connection string in tyk-data-plane
2. Suggest a few variables about how to get MDCB connection string dependent on service type

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] Documentation updates or improvements.

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [ ] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
     - [ ] I have manually updated the README(s)/documentation accordingly.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tyk-control-plane/templates/NOTES.txt [hunk 1]
```diff
@@ -60,14 +57,41 @@ If the worker gateway will be deployed via Helm, tyk-data-plane chart helps to f
...
+    --from-literal "groupID=$GROUP_ID" \
+    --namespace tyk-dp
+
+3- Refer this Kubernetes secret (tyk-data-plane-details) while installing worker gateways through `global.remoteControlPlane.useSecretName` in tyk-data-plane chart.
+
+4- Set `global.remoteControlPlane.connectionString`, `global.remoteControlPlane.useSSL` and `global.remoteControlPlane.sslInsecureSkipVerify` in tyk-data-plane chart to access MDCB service.
 
-3- Refer this Kubernetes secret (tyk-data-plane-details) while installing worker gateways through `global.remoteControlPlane.useSecretName`
-in tyk-data-plane chart.
+If data plane is deployed in the same cluster, it can be accessed via this connection string:
```
Comment (komalsukhani) [inline tyk-control-plane/templates/NOTES.txt:74]:
```suggestion
If data plane is deployed in the same cluster, MDCB can be accessed via this connection string:
```

File: tyk-control-plane/templates/NOTES.txt [hunk 2]
```diff
@@ -60,14 +57,41 @@ If the worker gateway will be deployed via Helm, tyk-data-plane chart helps to f
...
+{{- $tykMDCBSvc := printf "mdcb-svc-%s" (include "tyk-mdcb.fullname" (index .Subcharts "tyk-mdcb")) -}}
+{{- $tykMDCBSvcPort := (include "mdcb.svcPort" (index .Subcharts "tyk-mdcb")) }}
+    export MDCB_CONNECTIONSTRING="{{ $tykMDCBSvc }}.{{ .Release.Namespace }}.svc:{{ $tykMDCBSvcPort }}"
+
+If data plane is not deployed in the same cluster as control plane, get the connection string according to how MDCB service is exposed.
+
+{{- if contains "NodePort" (index .Values "tyk-mdcb" "mdcb" "service" "type") }}
+Node Port:
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ $tykMDCBSvc }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
```
Comment (komalsukhani) [inline tyk-control-plane/templates/NOTES.txt:86]:
```suggestion
  export NODE_IP=$(kubectl get nodes -o jsonpath="{.items[0].status.addresses[0].address}")
```

File: tyk-control-plane/templates/NOTES.txt [hunk 3]
```diff
@@ -60,14 +57,41 @@ If the worker gateway will be deployed via Helm, tyk-data-plane chart helps to f
...
+  export MDCB_CONNECTIONSTRING="$NODE_IP:$NODE_PORT"
+{{- else if contains "LoadBalancer" (index .Values "tyk-mdcb" "mdcb" "service" "type") }}
+TCP LoadBalancer:
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ $tykMDCBFullName }}'
```
Comment (komalsukhani) [inline tyk-control-plane/templates/NOTES.txt:91]:
```suggestion
           You can watch the status of service by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ $tykMDCBSvc}}'
```

File: tyk-control-plane/templates/NOTES.txt [hunk 4]
```diff
@@ -60,14 +57,41 @@ If the worker gateway will be deployed via Helm, tyk-data-plane chart helps to f
...
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ $tykMDCBSvc }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
+  export MDCB_CONNECTIONSTRING=$SERVICE_IP:{{ index .Values "tyk-mdcb" "mdcb" "service" "port" }}
```
Comment (komalsukhani) [inline tyk-control-plane/templates/NOTES.txt:93]:
```suggestion
  export MDCB_CONNECTIONSTRING=$SERVICE_IP:{{ $tykMDCBSvcPort }}
```

**Review by komalsukhani — CHANGES_REQUESTED** [review]

**Comment by caroltyk** [comment]
changes applied

**Review by komalsukhani — APPROVED** [review]

**Pull request merged**